Px.Utils/Validation/ContentValidation/ContentValidationFunctions.cs

```diff
@@ -0,0 +1,973 @@
+﻿using PxUtils.PxFile;
+using PxUtils.Validation.SyntaxValidation;
+
+namespace PxUtils.Validation.ContentValidation
+{
+    public delegate ValidationFeedbackItem[]? ContentValidationEntryFunctionDelegate(ValidationStructuredEntry entry, PxFileSyntaxConf syntaxConf, ref ContentValidationInfo info);
+    public delegate ValidationFeedbackItem[]? ContentValidationSearchFunctionDelegate(ValidationStructuredEntry[] entries, PxFileSyntaxConf syntaxConf, ref ContentValidationInfo info); 
+    
+    /// <summary>
+    /// Collection of functions for validating Px file metadata contents
+    /// </summary>
+    public class ContentValidationFunctions
+    {
+        public List<ContentValidationEntryFunctionDelegate> DefaultContentValidationEntryFunctions { get; }
+        public List<ContentValidationSearchFunctionDelegate> DefaultContentValidationSearchFunctions { get; }
+
+        /// <summary>
+        /// Constructor for <see cref="ContentValidationFunctions"/> class that contains default content validation functions/>
```

Comment:
"Constructor that contains default content validation functions" is enough, rest is redundant.